modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -2,62 +2,119 @@
 //
 // installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
 
-[id="configuring-host-network-interfaces-in-the-install-config-yaml-file_{context}"]
-= (Optional) Configuring host network interfaces in the `install-config.yaml` file
+:_content-type: PROCEDURE
+[id="configuring-host-network-interfaces-in-the-install-config.yaml-file_{context}"]
+= Optional: Configuring host network interfaces in the `install-config.yaml` file
 
-During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState. To use the `networkConfig` configuration setting, you must provide an NMState YAML configuration.
+During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState.
 
 [IMPORTANT]
 ====
-Before configuring host network interfaces, review the OpenShift Container Platform 4.10 release notes.
+Host network interface configuration is set during the installation and provisioning process and includes advanced options, such as setting static IPs per host. Before configuring host network interfaces, review the {product-version} 4.10 release notes. 
 ====
 
-See link:https://nmstate.io/examples.html#interfaces-ethernet[NMState] for additional examples of the NMState syntax.
+.Prerequisites
 
+* For each node being configured with a static IP, you must configure a `PTR` record containing a valid host name including the specific IP associated with that node.
+
+.Procedure
+
+. Optional: Consider saving the `networkConfig` YAML syntax to a file.
+
+.. Test the `networkConfig` YAML syntax by using the NMState command-line interface. Test the `networkConfig` YAML syntax before including it in the `install-config.yaml` file, because the installation program will not check the NMState YAML syntax. 
+
+.. Test the syntax by running the following command:
++
+.Example
+[source,terminal]
+----
+$ nmstatectl gc `<yaml-config>` 
+----
+
+[NOTE]
+====
+Errors in the YAML syntax might result in a failure to apply the network configuration. Maintain the valid YAML syntax when applying changes using Kubernetes NMState after deployment or when expanding the cluster.
+====
+
+. To use the `networkConfig` configuration setting, create and save an NMState configuration as a YAML file, as shown in the following example:
++
 .Example
-[source,yaml]
+[source,terminal]
 ----
   hosts:
         - name: openshift-master-0
           role: master
           bmc:
-            address: redfish+http://<out-of-band-ip>/redfish/v1/Systems/
-            username: <user>
-            password: <password>
+            address: redfish+http://_<out_of_band_ip>_/redfish/v1/Systems/
```

Comment:
(For this spot and all of the similarly-added underscores throughout)

If you're wanting these to be italicized (per [RH SSG](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values)), you'll need to add `[subs="+quotes"]` to the block as well, otherwise it will render the underscores verbatim in the built version. However, while that's the general CCS guideline, in the openshift-docs repo we have yet to adopt that guideline, so we haven't been italicizing user-replaced values.